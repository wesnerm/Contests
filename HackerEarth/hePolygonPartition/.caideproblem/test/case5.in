80
237 228
523 337
97 354
543 981
637 654
38 560
739 719
158 848
634 492
783 243
623 567
631 918
50 290
400 612
2 20
89 283
174 870
656 362
234 425
385 73
281 747
756 725
986 385
430 293
719 861
775 938
318 818
529 671
598 606
286 808
954 932
978 616
564 755
385 869
63 797
614 209
107 710
962 877
812 906
932 666
860 106
893 142
246 151
966 356
341 986
765 628
138 577
119 349
769 139
160 231
1000 561
230 923
247 565
831 609
856 653
860 692
675 621
949 848
39 836
370 413
790 28
528 78
31 986
71 530
590 612
107 672
745 700
666 299
877 970
896 681
321 356
501 688
125 72
954 517
629 272
310 933
770 748
442 972
32 151
891 261