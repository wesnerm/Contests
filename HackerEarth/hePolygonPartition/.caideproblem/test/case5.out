20
2 29 65
4 27 60 9 22
4 53 2 55 33
4 19 18 77 30
4 66 71 68 56
4 3 24 40 45
4 47 36 32 12
4 72 28 57 5
4 1 23 70 76
4 52 43 10 26
4 64 20 51 78
4 6 62 74 4
4 13 80 39 8
4 50 49 58 17
4 79 61 69 35
4 59 73 41 38
4 16 44 25 37
4 15 42 31 63
5 48 75 54 67 34
5 21 14 11 46 7
